platform/security/common/src/main/java/org/codice/ddf/security/common/Security.java

```diff
@@ -178,10 +191,18 @@ public final boolean javaSubjectHasAdminRole() {
    * will not change between calls.
    *
    * @return system's {@link Subject} or {@code null} if unable to get the system's {@link Subject}
+   * @throws SecurityException if a security manager exists and the {@link
+   *     javax.security.auth.AuthPermission AuthPermission("getSystemSubject")} or {@link
+   *     javax.security.auth.AuthPermission AuthPermission("getSubject")} permissions are not
+   *     authorized
    */
   @Nullable
   public synchronized Subject getSystemSubject() {
+    final java.lang.SecurityManager security = System.getSecurityManager();
 
+    if (security != null) {
+      security.checkPermission(Security.GET_SYSTEM_SUBJECT_PERMISSION);
```

Comment:
ℹ️ For when we stop embedding, this is the new permission we should check for in addition to the `"getSubject"` one instead of all the property and low level files access.